.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,14 +5,15 @@
 - Note: do not change the below
 ```
 
--  ##### Owner:
+- ##### Owner:
 
 - [ ] JIRA_ID included in all the commit messages
 - [ ] PR title is in the format JIRA_ID:Title
 - [ ] Rebase the branch with upstream
 - [ ] Squashed all commits into a single commit
 - [ ] Code passes ESLint tests
 - [ ] Added Tests
+- [ ] Ran cypress tests locally
```

Comment:
@SameeraPriyathamTadikonda  Let's mention the browser name as firefox here

Reply:
Done